x/fungible/keeper/gas_stability_pool.go

```diff
@@ -0,0 +1,99 @@
+package keeper
+
+import (
+	"math/big"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/zeta-chain/zetacore/x/fungible/types"
+)
+
+// EnsureGasStabilityPoolAccountCreated ensures the gas stability pool account exists
+func (k Keeper) EnsureGasStabilityPoolAccountCreated(ctx sdk.Context, chainID int64) {
+	address := types.GasStabilityPoolAddress(chainID)
+
```

Comment:
Should we add a check here , to ensure the chain ID is supported .

Reply:
As per Panruo comment, we remove this and use a single account for gas stability pool